src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/HealthCertifiedPerson.swift

```diff
@@ -155,8 +165,10 @@ class HealthCertifiedPerson: Codable, Equatable {
 			} else {
 				vaccinationState = .completelyProtected(expirationDate: vaccinationExpirationDate)
 			}
-		} else {
+		} else if !vaccinationCertificates.isEmpty {
```

Comment:
If we have 2 'outdated' certificates (in 2+ years) this would show a partial vaccination, right? Shouldn't we add an additional test for validity here?
I don't now the details but this could be similar to revoked certificates… 🤷‍♂️

Reply:
Validity of certificates is currently completely ignored apart from just showing the date in the UI. I guess this will be handled with a later story/tech spec update.